R/infrastructure.R

```diff
@@ -121,6 +142,11 @@ use_rcpp <- function(pkg = ".") {
   ignore_path <- file.path(pkg$path, "src", ".gitignore")
   union_write(ignore_path, c("*.o", "*.so", "*.dll"))
 
+  if (use_inst) {
+    use_directory("inst/include/", pkg = pkg)
+    use_template("inst_include_template", "/src/Makevars")
```

Comment:
I think this behavior would be a bit friendlier:

1. Only apply the template if `src/Makevars` doesn't exist already, or
2. Amend the `Makevars` file to add `PKG_CPPFLAGS = ../inst/include` if it already exists.

I would prefer having behavior 1) in this PR so that we don't overwrite a users potentially hand-crafted / fully populated Makevars file; 2) is more of a 'nice-to-have'.